docs/superpowers/plans/2026-04-16-anaf-mcp.md

```
@@ -0,0 +1,2402 @@
+```
+
+In another shell, send an MCP initialize request via stdin (or just confirm the process stays alive waiting on stdin). Expected: process starts without crashing; ListTools would return 7 tools.
+
+A lightweight check — pipe a ListTools JSON-RPC request and grep for tool names:
+
+```bash
+echo '{"jsonrpc":"2.0","id":1,"method":"tools/list"}' | ANAF_CLIENT_SECRET=dummy node mcp/dist/server.js | grep -o 'anaf_[a-z_]*' | sort -u
+```
+
+Expected output (7 lines):
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)